odk1-src/central-forms.rst

```diff
@@ -39,34 +44,53 @@ You can either click on the :guilabel:`choose one` button to browse for your :fi
+
+   .. image:: /img/central-forms/draft-overview.png
+
+The **Draft Status** page gives insight into the current status of your Draft, and provides controls for managing it.
+
+On the left, you will find the Draft Checklist, which suggests the steps you might take before publishing your Draft. On the right are details about the currently uploaded Draft version of the Form, including its current version string, and actions you may take on the Draft:
+
+ - The :guilabel:`Upload new definition` button will allow you to upload a new Form definition, which will replace the current Draft version. When this happens, all test submissions will be erased. If you have uploaded Media Files, Central will attempt to preserve any that match the new definition.
+ - The :guilabel:`Publish Draft` button will publish the Draft, making it available for use according to the access rules you have specified on the :ref:`Form Access <central-projects-form-access>` tab on the Project. Any test submissions you have made will be erased.
+ - The :guilabel:`Abandon Draft` button will delete the Draft. When there is not yet a published version, this will delete the entire Form. If the Form has been published, only the Draft will be deleted.
 
 .. tip::
-  When a form is first created, none of the existing App Users on the project will be able to access it for download or submission. Once you are ready to allow App Users to access the form, use the Project :ref:`Form Access <central-projects-form-access>` tab.
+  When a form is first created, none of the existing App Users on the project will be able to access it for download or submission, even once the Form is published. Once you are ready to allow App Users to access the form, use the Project :ref:`Form Access <central-projects-form-access>` tab.
 
-.. _central-forms-checklist:
+.. _central-forms-draft-tabs:
 
-The Form Overview page
-----------------------
+Form Draft Tabs
+~~~~~~~~~~~~~~~
 
-Here, you can get a brief summary of the status of your form, and recommended next steps. You are automatically taken here when you upload a new form or click on the form name in the Form listing page. You can also get back here from other form-related pages by clicking the :menuselection:`--> Overview` tab below the name of the form.
+When you first create a new Form, the navigation tabs on the left will not be accessible. They pertain to the published version of the Form, and will become available once you publish your Draft. The tabs on the right, within the gray Draft section, relate to the Draft.
```

Comment:
s/gray/darker gray/. There are two grays, so we should specify a bit more.

Reply:
huh? there are?